### PR TITLE
add tool for releasing the repo

### PR DESCRIPTION
**Pull request by aranguyen**
add tool for releasing the repo

### CODE REVIEW

File: .bcr/source.template.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
   "integrity": "",
-  "strip_prefix": "rules_platform-0.0.0",
-  "url": "https://github.com/bazelbuild/rules_platform/releases/download/0.0.0/rules_platform-0.0.0.tar.gz"
+  "strip_prefix": "rules_platform-0.1.0",
+  "url": "https://github.com/bazelbuild/rules_platform/releases/download/0.1.0/rules_platform-0.1.0.tar.gz"
```
Comment (katre) [inline .bcr/source.template.json:4]: Is this the link to the artifact created and uploaded in distro/README.md step 5?

File: distro/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Releasing rules_platform
+
+1. Update version in distro/BUILD.bazel,
```
Comment (katre) [inline distro/README.md:3]: This is a good first step, but we should eventually be able to automate this.

File: .bcr/source.template.json (hunk 1 above)
Comment (aranguyen) [reply .bcr/source.template.json:4]: Yes it is.  I think once this PR is submitted, uploading to BCR would happen and we need the correct link to the artifact. Last time there was a complaint when I did not upload the artifact beforehand.

File: distro/README.md (hunk 2 above)
Comment (aranguyen) [reply distro/README.md:3]: That's the next thing I'm looking into. `rules_java` has that automation done but I don't know how to set that up yet.

**Review by katre — APPROVED** [review]

File: MODULE.bazel [hunk 3]
```diff
@@ -8,3 +8,6 @@ module(
 )
 
 bazel_dep(name = "bazel_skylib", version = "1.5.0")
+
+# Dev dependencies
+bazel_dep(name = "rules_pkg", version = "0.10.0", dev_dependency = True)
```
Comment (gregestren) [inline MODULE.bazel]: Does this work with the newest release 0.10.1? https://github.com/bazelbuild/rules_pkg/releases/tag/0.10.1

**Review by gregestren — APPROVED** [review]

File: MODULE.bazel (hunk 3 above)
Comment (aranguyen) [reply MODULE.bazel]: it should work with the latest version. Let me update. THanks!

**Pull request merged**